pyproject.toml

```diff
@@ -1,13 +1,29 @@
-[project]
+[tool.poetry]
 name = "octreelib"
-dynamic = ["dependencies"]
-version = "0.0.5"
+version = "0.0.0"
+description = "Library for representing point clouds as OcTrees in SLAM"
+authors = [
+  "Kiselyov Mikhail <email@email.email>",
+  "Pavel Mokeev <pav3l.mokeev@gmail.com>"
+]
+license = "APACHE"
+readme = "README.md"
+homepage = "https://github.com/prime-slam/"
+repository = "https://github.com/prime-slam/octreelib"
+classifiers = [
+  "License :: OSI Approved :: Apache Software License",
+  "Intended Audience :: Developers",
+  "Topic :: Software Development :: Libraries :: Python Modules",
+]
```

Comment:
I think we also need to exclude tests from since it is a release package. 
Maybe we should use something like
```py
exclude = [
  "test"
]
```